docs/v3_services.rst

```diff
@@ -3,19 +3,20 @@ SecureDrop V3 Onion Services
 
 .. important::
 
-   SecureDrop instances must migrate to v3 onion services and Ubuntu 20.04
+   SecureDrop instances must migrate to v3 onion services and Ubuntu 20.04 (Focal)
    before April 30, 2021.
```

Comment:
This will only be released to the stable version of the docs after April 30, so I think we could be even more explicit here and just say that the Source Interface of any SecureDrop still running Xenial/v2 was disabled after April 30.

Reply:
Alternatively, perhaps we can phase this section of the docs out entirely with 2.0.0, and consolidate anything we still need from it into the backup/migration procedures.